core/config/docs/core.toml

```
@@ -291,6 +291,11 @@ ForceRedirect = false # Default
 # ListenIP specifies the IP to bind the HTTPS server to
 ListenIP = '0.0.0.0' # Default
 
+# Job Distributor stores the configuration for Job Distributor
+[JobDistributor]
+# NopFriendlyName is a friendly name set by the NOP to be shown in Job Distributor.
+NopFriendlyName = "" # Default
```

Please include an example since there is no default
```suggestion
NopFriendlyName = "" # Example
```

---

Included here: 6fd4238dd77d62de36234c0fc7a8c7faaf4c034d